packages/calypso-build/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@automattic/calypso-build",
+	"version": "1.0.0",
+	"description": "Shared build configurations.",
+	"keywords": [
+		"wordpress"
+	],
+	"author": "Automattic",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
```

Comment:
Should we set the package `private: true` and/or remove this bit until the package is ready? We're planning to build this package up more before publishing, correct?

Reply:
> Should we set the package `private: true` and/or remove this bit until the package is ready? 

Yep, makes a ton of sense!

> We're planning to build this package up more before publishing, correct?

Definitely :+1:

Reply:
bc0c51ac9b